src/main/resources/bidder-config/adf.yaml

```diff
@@ -3,11 +3,7 @@ adapters:
     endpoint: https://adx.adform.net/adx/openrtb
     aliases:
       adform:
-        usersync:
-          cookie-family-name: adform
-          redirect:
-            url: https://cm.adform.net/cookie?redirect_url={{redirect_url}}
-            uid-macro: '$UID'
+        enabled: false
```

Comment:
We don't add `enabled: false` for new aliases.
For this case we haaven't had this before, no need now